src/main/scala/com/evolutiongaming/util/dispatchers/Instrumented.scala

```diff
@@ -69,18 +69,20 @@ object Instrumented {
     def metrics(id: String, name: String, registry: MetricRegistry): Instrument = {
       val queue = registry histogram s"$name.queue"
       val run = registry histogram s"$name.run"
-      val workers = registry counter s"$name.workers"
+      val workersCurrent = registry counter s"$name.workers.current" // current value, very volatile
+      val workers = registry histogram s"$name.workers"              // aggregated metrics
 
       () => {
         val created = Platform.currentTime
         () => {
           val started = Platform.currentTime
           queue update started - created
-          workers.inc()
+          workersCurrent.inc()
+          workers.update(workersCurrent.getCount)
```

Comment:
what kind of information this will provide ?

Reply:
currently workers is just counter, and all we get once per minute, the value at the moment of measurement
this value changes ~200k per minute, so we just get one value of 200k and the rest is unknown
histogram accumulates all the values and shows things like mean, 99[9]th percentile, max value etc

Reply:
then I suppose we should just have `var counter` and use it for histogram and drop the `.inc` / `.dec` calls for good.
What do you think ?

Reply:
not sure about var
Counter is a wrapper around java.util.concurrent.atomic.LongAdder which is designed to be safe in multi-thread use and extremely performant

https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/atomic/LongAdder.html
"This class is usually preferable to AtomicLong when multiple threads update a common sum that is used for purposes such as collecting statistics, not for fine-grained synchronization control. Under low update contention, the two classes have similar characteristics. But under high contention, expected throughput of this class is significantly higher, at the expense of higher space consumption."

Reply:
why do we need workersCurrent metric? if LongAdder is so performant we can use it directly to maintain current number of active runnables